app/controllers/admin_controller.rb

```diff
@@ -1,7 +1,7 @@
 class AdminController < ApplicationController
 
   def index
-    
+    @top_cust = Customer.top_five_cust
```

Comment:
Does this break MVC? I think just basic AR queries like all, find, etc should go in the controller. We could call Customer.top_five_cust in the view instead.

Reply:
Agreed, this should not be called in the controller

Reply:
I see! My understanding was that querying shouldn't happen in the view, but instead the Controller should pass it to the view?

Reply:
To further clarify our decision, non-basic AR queries are called in the `Customer` model inside the `top_five_cust` method so that they are not living in the `Admin` controller, and then, the controller can use the method to query for the top five customers and send to the view.

Reply:
I said "our" decision, but I was navigating so don't want to throw @BLowrey24 under the bus HAHA

Reply:
Agreeing to disagree! Seems like a small thing for us to get hung up on for too long.